pkgs/by-name/xa/xavs2/package.nix

```diff
@@ -0,0 +1,70 @@
+{
+  lib,
+  fetchFromGitHub,
+  gitUpdater,
+  nasm,
+  stdenv,
+  testers,
+  validatePkgConfig,
+  pkgsBuildBuild,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "xavs2";
+  version = "1.4";
+
+  src = fetchFromGitHub {
+    owner = "pkuvcl";
+    repo = "xavs2";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-4w/WTXvRQbohKL+YALQCCYglHQKVvehlUYfitX/lLPw=";
+  };
+
+  env.NIX_CFLAGS_COMPILE = toString [
+    # Fixes compile error on darwin
```

Comment:
Nit:

```suggestion
  env.NIX_CFLAGS_COMPILE = toString (lib.optionals stdenv.isDarwin [
    # Fixes compile error on darwin
```

Reply:
Are you sure this is Darwin and not Clang?

Reply:
Did not check that.

Reply:
Is a clang thing and also zig.